origin_ui/src/app/(dashboard)/config/page.tsx

```diff
@@ -35,100 +35,21 @@ import {isLoggedIn} from "@/helpers/login";
 
 type duration = number | `${number}${"ns" | "us" | "µs" | "ms" |"s" | "m" | "h"}`;
 
-interface Config {
-    ConfigDir: string;
-    Debug: boolean;
-    TLSSkipVerify: boolean;
-    IssuerKey: string;
-    Transport: {
-        DialerTimeout: duration;
-        DialerKeepAlive: duration;
-        MaxIdleConns: number;
-        IdleConnTimeout: duration;
-        TLSHandshakeTimeout: duration;
-        ExpectContinueTimeout: duration;
-        ResponseHeaderTimeout: duration;
-    }
-    Federation: {
-        DiscoveryUrl: string;
-        TopologyNamespaceUrl: string;
-        DirectorUrl: string;
-        NamespaceUrl: string;
-        JwkUrl: string;
-    }
-    Client: {
-        StoppedTransferTimeout: number;
-        SlowTransferRampupTime: number;
-        SlowTransferWindow: number;
-        DisableHttpProxy: boolean;
-        DisableProxyFallback: boolean;
-        MinimumDownloadSpeed: number;
-    }
-    DisableHttpProxy: boolean
-    DisableProxyFallback: boolean;
-    MinimumDownloadSpeed: number;
-    Origin: {
-        Url: string;
-        ExportVolume: string;
-        NamespacePrefix: string;
-        Multiuser: boolean;
-        UseCmsd: boolean;
-        UIPasswordFile: string;
-        SelfTest: boolean;
-    };
-    Director: {
-        DefaultResponse: string;
-        MaxMindKeyFile: string;
-        GeoIPLocation: string;
-    };
-    Registry: {
-        DbLocation: string;
-    };
-    Server: {
-        TLSCertificate: string;
-        TLSCACertificateFile: string;
-        TLSCACertificateDirectory: string;
-        TLSCAKey: string;
-        TLSKey: string;
-        Port: number;
-        Address: string;
-        ExternalAddress: string;
-        Hostname: string;
-        IssuerJwks: string;
-    };
-    OIDC: {
-        ClientIDFile: string;
-        ClientSecretFile: string;
-        DeviceAuthEndpoint: string;
-        TokenEndpoint: string;
-        UserInfoEndpoint: string;
-    };
-    Xrootd: {
-        Port: number;
-        RunLocation: string;
-        RobotsTxtFile: string;
-        ScitokensConfig: string;
-        Mount: string;
-        MacaroonsKeyFile: string;
-        Authfile: string;
-        ManagerHost: string;
-        SummaryMonitoringHost: string;
-        DetailedMonitoringHost: string;
-        LocalMonitoringHost: string;
-        Sitename: string;
-    };
-    Monitoring: {
-        DataLocation: string;
-        PortLower: number;
-        PortHigher: number;
-    };
+type Config = {
+    [key: string]: ConfigValue
 }
 
-function sortConfig (a: any, b: any) {
-    if(typeof a[1] == 'object' && typeof b[1] != 'object'){
+type ConfigValue = Config | string | number | boolean | null | string[] | number[] | duration
```

Comment:
Thanks for making this change! It will be greatly improve the robustness of the way we displaying config values